device/ragile/x86_64-ragile_ra-b6920-4s-r0/plugins/ssd_util.py

```diff
@@ -5,6 +5,7 @@
 from sonic_platform_base.sonic_ssd.ssd_base import SsdBase
 from subprocess import Popen, PIPE
 from re import findall
+from shlex import split
 from os.path import exists
 
 INNODISK = "iSmart -d {}"
```

Comment:
>INNODISK [](http://example.com/codeflow?start=0&length=8)

Better to construct args array directly instead of constructing a full command string and split into args array.

Reply:
Fixed.